contracts/src/token/erc721/mod.rs

```diff
@@ -483,8 +483,6 @@ impl IErc721 for Erc721 {
         to: Address,
         token_id: U256,
     ) -> Result<(), Error> {
-        // TODO: Once the SDK supports the conversion,
-        // use alloy_primitives::bytes!("") here.
```

Comment:
Is it actually significantly better that it's worth tracking? I think the way it is today is probably ok, but happy to hear thoughts if I'm missing something(e.g. gas saving)

Reply:
I do not think so.

Reply:
Not significantly better. I suspect would be zero difference in gas:) Do we have a zero todo's policy in code?:D

Reply:
No, I don't think we should aim for that. Just maybe before each audit we make sure to clean things and move outstanding TODOs to issues? Since we know otherwise they will become findings in the report